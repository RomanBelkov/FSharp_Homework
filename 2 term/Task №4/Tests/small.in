100
m o eajwemweootoclcs
welcoooooooooome to code jam
welcome to code jam
j e d mecmt ee mx
welllcooomeee ttto coddde jam
weelcooome to cooddde jjam
welcomem  to codede jjam
rawpt wtwmeoloji
welco  cme two mecode de jam
welccome too code jam
welcccomeee ttto     codes jam
welccomem to e tc ocde jammm
swlmacclc ajoan
wellcocome t oo ccodde  jjaamm
welelocoem e to codeode jaamn
hlelocjlom emoo
weelcommeome  too dcoded ejjam
rcwwoe kooe womj mc g
bweelcolmwe to ciode jam
wweellccoommee to code qps jam
wwelcomme to  coodde jam
welchome tgo     ccqconxde jam
wwelccmome tto ccodee ajam
vemdw  aeoo ewok
wellcomme toto coode ejaam
welcomme tot  code  jam
o tmeanae looeg
weeclccome to code jamm
welcome eto codee  jam
wwelcomoee teo  ococdde j aamm
wooetommteo wotn
ret lrwm eewoeah
welcome to c ode jamm
welcome to codejam
welcome to code jam
welcome tto code jam
welcome to code jam
ctocoeecodej  lmmmdlaalcor
weelcokwme to cc oodmee ejam
umo  o tmdoowolomx
welcomee too coode jam
wwwelllcome tto code jjjjjamm
weblcomerq tqho xgcsgobde njam
welcome to ocodoe  jam
welllcoome to codde jam
aj  ljemioonmmomw
weelcomme  to codde jam
welccomoe  ttoo o codde jjaam
zoat  dseet  eaada
welcome to code jamjamjamjam
elcomew elcome to code jam
wellcome  to code jamm
wellcomeo me tol coede  ja m
welcoomoe  tto cooede jjam
cacommam jecjtet l r
weelcomme t ot codeo je aamm
welccowme tote c odceod j aamm
you are welcomed to code jam
j licdectfejwmcocq
wellocomme  too coode jam
jjodcd  o cjwjboojjlmejwdljenp
welcomme to coded jaaam
wwelcomoee to o codoee  jamm
oc  kcolomlocc cel
nm xtcdontodolomce q
f o eca cmwcloj eea a m cmlw
wcecweeoe lmlc b
weeclcome tto code jammm
welecoome ettoo  code dj jamm
q welcome to code jam p
z
w elcotme to ocode mjjacmcl
welcome to code jam
wellcome  to code jam
welcomme to code jam
wellcmo meo to cdode jam
welcome to code jam
gwde lcomlmoe ateoo acode jatm
hwtotdme  ew joodtteywek
welcoomee to cdode jam
wwelcomle emt oco cocodea jam
welcome t ot codee jam
weelcomoee e to coode ejjamm
wwelcome to  codec djeaojmam
wwwwwwwelcome to code jjjam
wwwelclocome  too ccodee jamam
weelcomoe tteot  code jamam
wwlelcomome to  o dco de ejaam
not really a welcome message
welcome to  code  jamm
wweelcome to codee jamm
w
welcome tto cooddee jam
qojoleeaxo etwaeomwoax
wljeowlcmomae to tcodo e jam
vecoeltmcattoyb
vcewt mo roomoowtajtjcmjdmwta
weulcomed tmo cocde cojam
wwelcome et o  code jamm
welcome to code jam
