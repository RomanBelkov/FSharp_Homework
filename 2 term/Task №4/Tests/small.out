Case #1: 0000
Case #2: 0010
Case #3: 0001
Case #4: 0000
Case #5: 0243
Case #6: 0072
Case #7: 0012
Case #8: 0000
Case #9: 0004
Case #10: 0004
Case #11: 0135
Case #12: 0012
Case #13: 0000
Case #14: 0768
Case #15: 0024
Case #16: 0000
Case #17: 0096
Case #18: 0000
Case #19: 0002
Case #20: 0256
Case #21: 0032
Case #22: 0015
Case #23: 0032
Case #24: 0000
Case #25: 0048
Case #26: 0008
Case #27: 0000
Case #28: 0008
Case #29: 0004
Case #30: 0064
Case #31: 0000
Case #32: 0000
Case #33: 0002
Case #34: 0000
Case #35: 0001
Case #36: 0002
Case #37: 0001
Case #38: 0000
Case #39: 0016
Case #40: 0000
Case #41: 0008
Case #42: 0180
Case #43: 0001
Case #44: 0002
Case #45: 0012
Case #46: 0000
Case #47: 0016
Case #48: 0320
Case #49: 0000
Case #50: 0020
Case #51: 0001
Case #52: 0008
Case #53: 0020
Case #54: 0032
Case #55: 0000
Case #56: 0016
Case #57: 0008
Case #58: 0001
Case #59: 0000
Case #60: 0032
Case #61: 0000
Case #62: 0006
Case #63: 0096
Case #64: 0000
Case #65: 0000
Case #66: 0000
Case #67: 0000
Case #68: 0012
Case #69: 0096
Case #70: 0001
Case #71: 0000
Case #72: 0002
Case #73: 0001
Case #74: 0004
Case #75: 0002
Case #76: 0002
Case #77: 0001
Case #78: 0004
Case #79: 0000
Case #80: 0004
Case #81: 0012
Case #82: 0002
Case #83: 0080
Case #84: 0016
Case #85: 0021
Case #86: 0576
Case #87: 0024
Case #88: 0048
Case #89: 0000
Case #90: 0008
Case #91: 0016
Case #92: 0000
Case #93: 0016
Case #94: 0000
Case #95: 0001
Case #96: 0000
Case #97: 0000
Case #98: 0001
Case #99: 0008
Case #100: 0001
